pkg/plugins/plugins.go

```diff
@@ -38,6 +38,7 @@ type Plugin struct {
 	BaseURL string
 
 	Renderer       pluginextensionv2.RendererPlugin
+	Sanitizer      pluginextensionv2.SanitizerClient
```

Comment:
Is this really needed? It doesn't seem used and the sanitizing function is available through the `Renderer` plugin above.